deployment/sandbox-v2/docs/idacert.md

```diff
@@ -20,212 +21,126 @@
 	  "version": "string"
 	}
 	```
-  * 2. Get the ROOT certificate data from the below URL, Copy it and use it for certificate exchange in the next step.
-	```https://minibox.mosip.net/idauthentication/v1/internal/getCertificate?applicationId=ROOT```
-
-  * 3. Upload ROOT certificate from the above request in the below SWAGGER URL:- ```https://minibox.mosip.net/v1/partnermanager/swagger-ui.html#/Partner%20Service%20Controller/uploadCACertificateUsingPOST``` Partner_Service_Controller --> /partners/certificate/ca/upload --> with below request
+  * 2. Get the certificate data from the below URL.The used application id = IDA ad reference id =CRED_SERVICE  
```

Comment:
Get the certificate data from the URL mentioned below. The application id used is IDA and reference id =CRED_SERVICE 

(Please check if application id has and underscore and the same for reference id)

Reply:
I see it written as appId below... so, we can mention Application ID (appId) for clarity